### PR TITLE
coova-chilli: switch to procd

### PR DESCRIPTION
**Pull request by teslamint**
Maintainer: @kaloz 
Compile tested: N/A
Run tested: mvebu/generic, Turris Omnia, OpenWrt 15.05 r47055

Description:
able to launch (multiple) chilli instance with procd

### CODE REVIEW

**Comment by teslamint** [comment]
ping?

**Comment by yousong** [comment]
Please also updating `PKG_HASH` to use sha256sum.  This should pass the travis check warning

```make[1]: Entering directory `/tmp/tmp.47QtN2EV7R'
WARNING: PKG_MIRROR_HASH is missing, set to 89c9b313881c658a0f6b91329a78bb1a0151878b19bc99b315976081c6355557
make[2]: Entering directory `/home/travis/build/openwrt/packages/net/coova-chilli'
```

File: net/coova-chilli/files/chilli.init [hunk 1]
```diff
@@ -1,61 +1,106 @@
...
+	procd_set_param command /usr/sbin/chilli
+	procd_set_param file /etc/config/chilli
+	procd_append_param command --fg
+	procd_append_param command --conf "${base}.conf"
+	procd_append_param command --pidfile "${base}.pid"
+	procd_append_param command --cmdsocket "${base}.sock"
+	procd_append_param command --unixipc "${base}.ipc"
+	procd_set_param respawn
+	procd_set_param stdout 1
+	procd_set_param stderr 1
```
[the remark this answers is not in the document]
Comment (teslamint) [reply net/coova-chilli/files/chilli.init]: Not before version 1.3.1.4.

**Comment by yousong** [comment]
Hi @teslamint , as @kaloz has been absent for a while, are you interested in taking the maintainership of the package?  If that's the case, please adding an additional commit changing the `PKG_MAINTAINER` field.

**Comment by teslamint** [comment]
@yousong Sure.

File: net/coova-chilli/files/chilli.init [hunk 2]
```diff
@@ -1,61 +1,84 @@
...
+
+reload_chilli() {
+	local pid
+	local cfg="$1"
+	local base="/var/run/chilli_${cfg}"
+	if [ -f "${base}.pid" ]; then
+		pid="$(cat "${base}.pid")"
+		[ -f "/var/run/chilli.${pid}.cfg.bin" ] && rm -f "/var/run/chilli.${pid}.cfg.bin"
+		chilli_query -s "${base}.sock" reload
+		sleep 1
```
Comment (yousong) [inline net/coova-chilli/files/chilli.init]: is this `sleep 1` really necessary?  We should avoid such magic delay period when possible

**Comment by yousong** [comment]
Thank you for the efforts.  and sorry for the delay.

**Comment by teslamint** [comment]
My pleasure.

**Pull request merged**